docker-compose.yml

```diff
@@ -1,27 +1,40 @@
-version: '2'
+version: '3.8'
+
 services:
   postgres:
-    image: postgres:9.5-alpine
+    image: postgres:12-alpine
+    hostname: stringer-postgres
     restart: always
+    networks:
+      - stringer
     volumes:
-      - ~/stringer:/var/lib/postgresql/data
+      - ./postgres:/var/lib/postgresql/data
     environment:
-      - POSTGRES_PASSWORD=super_secret_password
+      - POSTGRES_PASSWORD=<PLEASE_ENTER_YOUR_PASSWORD>
       - POSTGRES_USER=db_user
       - POSTGRES_DB=stringer
 
   web:
-    image: mockdeep/stringer
+    image: mockdeep/stringer:latest
```

Comment:
these images should actually move to `stringerrss/stringer` at some point. Not sure if now is the time, since we're currently [unable to publish the image just yet](https://app.circleci.com/pipelines/github/stringer-rss/stringer/1003/workflows/493f8194-6145-4321-b82e-a8b56056ebd0/jobs/1013).